static/app/utils/fields/index.ts

```diff
@@ -1171,26 +1190,11 @@ const REPLAY_FIELD_DEFINITIONS: Record<ReplayFieldKey, FieldDefinition> = {
     kind: FieldKind.FIELD,
     valueType: FieldValueType.STRING,
   },
-  [ReplayFieldKey.RELEASES]: {
-    desc: t('Releases this Replay spans across'),
-    kind: FieldKind.FIELD,
-    valueType: FieldValueType.STRING,
-  },
```

Comment:
Same as above.  Do we need to keep this to support `releases` in addition to `release`?